.github/workflows/dependency-review.yml

```diff
@@ -5,6 +5,10 @@ on:
   workflow_dispatch:
     inputs: {}
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

Comment:
@WillDaSilva So `head_red` is available in PRs and commits, but not workflow dispatch, hence the fallback?